OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -872,6 +872,23 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					}
 				}
 
+				if (engineVersion < 20150427)
+				{
+					if (node.Key.StartsWith("ProductionQueue@") || node.Key.StartsWith("ClassicProductionQueue@"))
+						node.Value.Nodes.RemoveAll(n => n.Key == "RequireOwner");
+
+					if (node.Key == "Buildable")
+					{
+						var removed = node.Value.Nodes.RemoveAll(n => n.Key == "Owner");
+						if (removed > 0)
+						{
+							Console.WriteLine("The 'Owner' field has been removed.");
+							Console.WriteLine("Please use prerequisites instead.");
+							Console.WriteLine("");
```

Comment:
Minor style nit - an empty string is not needed here.
The method has an override that takes no arguments and just prints an empty line.